auth-web/src/util/constants.ts

```diff
@@ -59,6 +59,7 @@ export enum Role {
     ViewAllTransactions = 'view_all_transactions',
     ManageEft = 'manage_eft',
     EftRefund = 'eft_refund',
+    EftRefundApprover = 'eft_refund_approver',
```

Comment:
deleted my comment, will have to create this role

Reply:
![image](https://github.com/user-attachments/assets/959dc75f-32db-473b-995f-3382ed1ed0db)

Reply:
done in all 3 environments